lab-kaylyn/app/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html ng-app="demoApp">
+<!-- allows angular (from entry.js) to bind itself to this HTML tag -->
+<!-- will not work without ng app -->
+<head>
+  <meta charset="utf-8">
+  <title>Cowsay Client</title>
+</head>
+<body>
+  <i class="fa fa-hand-rock-o"></i>
+  <i class="fa fa-hand-paper-o"></i>
+  <i class="fa fa-hand-scissors-o"></i>
+  <section ng-controller="CowsayController as cowsayCtrl" ng-init="cowsayCtrl.current='satanic'">
+    <h1>{{cowsayCtrl.title}}</h1>
+    <form ng-submit="cowsayCtrl.duplicate(cowsayCtrl.text)"
+    novalidate
```

Comment:
@kaylynyuh best to indent these over 1 tab (2 spaces) when adding new lines for attributes